common/app/routes/Settings/components/DangerZone.jsx

```diff
@@ -47,6 +47,11 @@ class DangerZone extends PureComponent {
   }
 
   render() {
+    const centerText = {
```

Comment:
Remove this styling, this should be rather handled in a dedicated styles file, for the component group. On that note, for this particular message this is an unneeded change.